core/lib/zksync_core_leftovers/src/temp_config_store/mod.rs

```diff
@@ -111,6 +111,7 @@ impl TempConfigStore {
             snapshot_recovery: self.snapshot_recovery.clone(),
             pruning: self.pruning.clone(),
             external_price_api_client_config: self.external_price_api_client_config.clone(),
+            consensus_config: None,
```

Comment:
could we use `config::read_consensus_config()` here?

Reply:
that's a different crate